lbrycrd/Dockerfile

```diff
@@ -25,7 +25,7 @@ COPY stuff/start.sh /usr/local/bin/start
 COPY stuff/docker-entrypoint.sh /usr/local/bin/docker-entrypoint
 COPY stuff/healthcheck.sh /usr/local/bin/healthcheck
 
-# USER lbrycrd
+USER lbrycrd
```

Comment:
I opted to run this as root and then a privilege de-escalation using `su -c {invocation} lbrycrd` so that it wouldn't matter what the user did to the file permissions we could re-assert them properly prior to execution.

Reply:
Ah that does make sense. I had a problem with it however:

```
su : must be run from a terminal
```

I really don't know what causes it though.

Reply:
I'm glancing around for any other potentially related issues to see what solutions may be for this however if you find one before I do feel free to tweak appropriately.

Reply:
So without "USER lbrycrd" every command is run as root, which means that if they don't use start.sh, things will run as root. This is probably wrong, so it would be fixed with an ENTRYPOINT (then nothing could run except for start.sh, which does the user descalation) But, if were to use an ENRTRYPOINT, I would want to make sure that we are not hiding any of parameters from the lbrycrd binary itself. That's one of the problems with wrapper scripts..

Reply:
This is just an idea, I haven't tested it, but can we have "USER lbrycrd" and then set the start.sh script as setuid root? That way nothing could run as root except for start.sh, which can fix permissions, then deescalate.

Reply:
or maybe add another script to stuff called fix_permissions.sh and set *that* script to setuid root.

Reply:
So the only problem is if the config's ownership is already set to Root and it needs to be executable by the user `lbrycrd` it would not have access.  You could give the `lbrycrd` user the ability to use `sudo` however that would be self-defeating, and at that point you might as well operate the application at the root level, to begin with within the container.

Reply:
I believe that the solution will have to likely reside within aiming for an alternative to `su -c` which operates the same and doesn't necessarily require more things to be installed if at all possible.  I've seen things like `su-exec` and whatnot but I've not tested them.